cicecore/cicedynB/dynamics/ice_dyn_vp.F90

```diff
@@ -87,7 +88,8 @@ module ice_dyn_vp
          reltol_andacc      ! relative tolerance for Anderson acceleration
 
       character (len=char_len), public :: &
-         precond        , & ! preconditioner for fgmres: 'ident' (identity), 'diag' (diagonal), 'pgmres' (Jacobi-preconditioned GMRES)
+         precond        , & ! preconditioner for fgmres: 'ident' (identity), 'diag' (diagonal), 'pgmres'
+                            ! (Jacobi-preconditioned GMRES)
```

Comment:
minor, but I'd prefer keeping the "pgmres' option on the same line as its description

Reply:
This is one the few things -Wall complained about.  Not sure what the "extended" line limit is, but some of these lines were "too long".  I'm happy to undo all of these or is it better to conform to what -Wall says?

Reply:
this was fixed.